modules/acquia_cms_headless/tests/src/Functional/DashboardApiDocumentTest.php

```diff
@@ -10,7 +12,9 @@
  * @group medium_risk
  * @group push
  */
-class DashboardApiDocumentTest extends DashboardWebDriverTestBase {
+class DashboardApiDocumentTest extends DashboardTestBase {
```

Comment:
Lets try to use a common naming convention for all headless test base instead of DashboardTestBase